fractal_tasks_core/tasks/projection.py

```diff
@@ -189,5 +142,5 @@ def projection(
 
     run_fractal_task(
```

Comment:
To discuss with @tcompa: When does `run_fractal_task` use the logger name? Is the ngio_logger also a reasonable choice here?